CHANGELOG.md

```diff
@@ -7,6 +7,7 @@
 ### What's new
 
 - **Added new `Fieldset` component** for grouping inputs.
+- **Added Split Header Button**
```

Comment:
Add more wording about how it's enabled. "Split Header Button" is not a component so the user would have no idea.